patches/chromium/feat_implement_unset_window_aspect_ratio_on_linux.patch

```diff
@@ -0,0 +1,49 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
The default should be to try upstream this patch, before we land this patch into `electron` can you submit upstream and see if it lands?